Remotion/Data/DomainObjects.UnitTests/SetupFixture.cs

```diff
@@ -15,87 +15,32 @@
-        var testDomainAgent = new DatabaseAgent(DatabaseTest.TestDomainConnectionString);
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TestDomain_TearDownDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TableInheritanceTestDomain_TearDownDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TestDomain_SetupDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_TableInheritanceTestDomain_SetupDB.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-        testDomainAgent.ExecuteBatchFile("Database\\DataDomainObjects_Patches.sql", true, DatabaseConfiguration.GetReplacementDictionary());
-
-        _standardMappingDatabaseAgent = new StandardMappingDatabaseAgent(DatabaseTest.TestDomainConnectionString);
-        string sqlFileName = StandardMappingTest.CreateTestDataFileName;
-        _standardMappingDatabaseAgent.ExecuteBatchFile(sqlFileName, true, DatabaseConfiguration.GetReplacementDictionary());
-        string sqlFileName1 = TableInheritanceMappingTest.CreateTestDataFileName;
-        _standardMappingDatabaseAgent.ExecuteBatchFile(sqlFileName1, true, DatabaseConfiguration.GetReplacementDictionary());
-
-        // We don't want the tests to initialize a default mapping; therefore, modify MappingConfiguration.s_fields.Current so that it will 
-        // throw when asked to generate a new MappingConfiguration.
-
-        var throwingMappingConfigurationContainer = new DoubleCheckedLockingContainer<IMappingConfiguration>(
-            () =>
-            {
-              throw new InvalidOperationException(
-                  "This test failed to setup the mapping configuration. Did you forget to derive from StandardMappingTest or to call base.SetUp?");
-            });
-        var fields = PrivateInvoke.GetNonPublicStaticField(typeof(MappingConfiguration), "s_fields");
-        Assertion.IsNotNull(fields);
-        _previousMappingConfigurationContainer = (DoubleCheckedLockingContainer<IMappingConfiguration>)PrivateInvoke.GetPublicField(fields, "Current");
-        PrivateInvoke.SetPublicField(fields, "Current", throwingMappingConfigurationContainer);
-      }
-      catch (Exception ex)
-      {
-        Console.WriteLine("SetUpFixture failed: " + ex);
-        Console.WriteLine();
-        throw;
-      }
+      LogManager.ResetConfiguration();
+      Assert.That(LogManager.GetLogger(typeof(LoggingClientTransactionListener)).IsDebugEnabled, Is.False);
     }
 
     [OneTimeTearDown]
     public void OneTimeTearDown ()
     {
-      var fields = PrivateInvoke.GetNonPublicStaticField(typeof(MappingConfiguration), "s_fields");
-      Assertion.IsNotNull(fields);
-      PrivateInvoke.SetPublicField(fields, "Current", _previousMappingConfigurationContainer);
```

Comment:
You dropped this. Why do we not need it?

Reply:
It is kind of pointless to set a public static field right before the test run ends anyways as this method is called after all the tests just before the shutdown of the test process.